torch/csrc/distributed/rpc/message.h

```diff
@@ -128,7 +128,8 @@ TORCH_API Message createExceptionResponse(const std::exception& e, int64_t id);
 TORCH_API Message
 createExceptionResponse(const std::string& exceptionStr, int64_t id);
 
-typedef torch::utils::Future<Message> FutureMessage;
+using FutureMessage = torch::utils::Future<Message>;
```

Comment:
Nit: Add a note that this is an internal type and the future sent back to users should be Future<IValue>?